README.md

```diff
@@ -32,14 +32,31 @@ Set name of your newly created app to your environment to make things simpler:
 export HEROKU_APP_NAME=app_name  # please replace with your app name
 ```
 
-Finally run the following command and follow the instructions. You will be
-prompted to login to your Salesforce sandbox. 
+#### Configuring Salesforce connection
+
+You can configure a Salesforce connection using the UI:
+
+```bash
+heroku addons:open herokuconnect -a "$HEROKU_APP_NAME"
+```
+
+It will open the Heroku Connect page. Click on Setup and Next, Next, Next.
+
+From the command line, run the following command and follow the instructions.
+You will be prompted to login to your Salesforce sandbox. 
 
 ```bash
 heroku connect:info -a "$HEROKU_APP_NAME"
 heroku connect:db:set --schema salesforce --db DATABASE_URL -a "$HEROKU_APP_NAME"
 heroku connect:sf:auth -a "$HEROKU_APP_NAME"
 sleep 3  # give heroku connect a second to connect
+```
+
+#### Importing Django Heroku Connect schema
+
+For both you should import the schema:
```

Comment:
I am not sure I am following context here. What is `both` here?

Reply:
Hummm I might have to add this `In both (interface or exclusive via terminal) you should import the schema:`. What do you think?

Reply:
@anapaulagomes yeah, now it makes sense. 👍